controlplane/test/apollo-federation.test.ts

```diff
@@ -22,7 +17,7 @@ describe('Apollo Federated Graph', (ctx) => {
 
   test('Should be able to create a Apollo Federated Graph', async (testContext) => {
     const { client, server } = await SetupTest(testContext, dbname);
-    
```

Comment:
I'm missing tests that do operations across namespaces. We have to ensure that we won't collide with other namspaces.

Reply:
Added tests